packages/timeline/src/MiniGantt.ts

```diff
@@ -75,10 +75,15 @@ export class MiniGantt extends SVGWidget {
     }
 
     extent() {
-        if (this.rootExtent) {
-            return [this.rootExtent[1], this.rootExtent[2]];
+        const extent = this.rootExtent ? [this.rootExtent[1], this.rootExtent[2]] : this.fullExtent();
+        if (extent[0] === extent[1]) {
+            const parser = d3TimeParse(this.timePattern());
+            const formatter = d3TimeFormat(this.timePattern());
+            const dt = parser(extent[0]);
+            extent[0] = formatter(new Date(dt.setFullYear(dt.getFullYear() - 1)));
+            extent[1] = formatter(new Date(dt.setFullYear(dt.getFullYear() + 2)));
```

Comment:
Shouldn't this be:
```
const dtHi = parser(extent[1]);
extent[1] = formatter(new Date(dtHi.setFullYear(dtHi.getFullYear() +1)));
```

Reply:
That conditional (`extent[0] === extent[1]`) evaluates to true when there's only one unique timestamp in the dataset.